lib/pf/cmd/pf/pfconfig.pm

```diff
@@ -0,0 +1,187 @@
+package pf::cmd::pf::pfconfig;
+=head1 NAME
+
+pf::cmd::pf::pfconfig add documentation
```

Comment:
Nope nope nope nope nope nope
Please change the template, tired to see this all over the code.